applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -55,7 +55,8 @@ def __init__(self, model, custom_settings):
             "analysis_type": "non_linear",
             "model_import_settings": {
                 "input_type": "mdpa",
-                "input_filename": "unknown_name"
+                "input_filename": "unknown_name",
+                "skip_timer": true
```

Comment:
note that this is not used, since we are not recursivly validating
@jginternational could you please add it to the problems in Structural?

Reply:
nevermind @jginternational , I made skipping the timer the default now, so you don't have to do anything
sorry for the noise